releases/release-1.29/release-notes/maps/pr-120825-map.yaml

```diff
@@ -1,7 +1,7 @@
 pr: 120825
 releasenote:
-  text: '`kubeadm`: allow deploying a kubelet that is 3 versions older than the version
-    of `kubeadm` (N-3). This aligns with the recent change made by SIG Architecture
+  text: '`kubeadm`: allowed deploying a kubelet that is 3 versions older than the
```

Comment:
text: '`kubeadm`: will now allow deploying a `kubelet` that is 3 versions older than the